src/common/types/threadedComments.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+export type ThreadedCommentStatus = 'open' | 'resolved';
```

Comment:
I recall a `deleted` status being added in the Annotations PR. Are the available statuses different across the various resources? Why?